### PR TITLE
Add draft content - "DDMoRe Language Community" and "...But I only use NONMEM"

### PR DESCRIPTION
**Pull request by MikeKSmith**
Added two new blog posts. 
DDMoRe Language Community - introduces the DDMoRe Language Community, its purpose, role and how it will interact with the DDMoRe Foundation. 
"...but I only use NONMEM" - to counter the argument that MDL isn't needed because most users only use NONMEM.

### CODE REVIEW

File: content/post/But_I_only_use_NONMEM.Rmd [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+"validated" and accepted by regulatory agencies. See 
+[here](http://www.slideshare.net/MikeKSmith/r-eliminating-fear-uncertainty-and-doubt)
+for more discussion on this topic. But if we follow that argument through,
+why aren't people insisting that we re-run NONMEM analyses using PROC NLMIXED?*
+`</end DIGRESSION>`
+
+### I share via publications
+Writing down the maths of a model is a great way to share the model between
+quantitative disciplines. It's a universal language and brings with it rigour
+and a lack of ambiguity. ***BUT*** while two modellers can agree on the maths
```
Comment (vjd) [inline content/post/But_I_only_use_NONMEM.Rmd:56]: Maths comes with assumptions, and most authors do their best to state these assumptions. Can you imagine the horror when the implemented math is different from the assumptions that the author was making?

File: content/post/But_I_only_use_NONMEM.Rmd [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+quantitative disciplines. It's a universal language and brings with it rigour
+and a lack of ambiguity. ***BUT*** while two modellers can agree on the maths 
+of a problem, the implementation may vary between two software tools such that
+it may be easy to implement in one tool, but very difficult (impossible?) in 
+another.
+
+## NONMEM doesn't do ***EVERYTHING***
+NONMEM and associated tools e.g. PsN have some limitations in available 
+functionality, for example optimal trial design. And while NONMEM ***can*** be
+used for prediction and simulation of new designs, regimens, scenarios there
```
Comment (vjd) [inline content/post/But_I_only_use_NONMEM.Rmd]: and scenarios,

File: content/post/But_I_only_use_NONMEM.Rmd [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+therapeutic drug models in their work or for predicting outcomes for patients.
+
+About 10 years ago, it would have been hard to imagine the case that S-Plus
+would be thoroughly supplanted by R, to the point where it ceased to be a 
+supported product by the company (Tibco) that bought it from Insightful. And yet, here
+we are. [R is rising in popularity as an analytical engine (often coupled with 
+other technologies)](http://blog.revolutionanalytics.com/popularity/). So to 
+dismiss open-source solutions as "irrelevant" may be regarded as a 
+short-sighted view.
+
```
Comment (vjd) [inline content/post/But_I_only_use_NONMEM.Rmd:99]: For a moment, just compare the growth our community in pharmacometrics made in comparison to other fields such as computational biology, genetics, ecology to name a few. The experts in these fields are scientists, similar to us, and not software engineers. Yet, they made massive strides in science mainly due to their adoption of open source methods and tools. Experts working in the industry put their developed tools and materials out in the open, a culture that pharmacometrics experts in the industry have yet to embrace after all these years. 

The fundamental requirement is a change in culture and thinking in our community. To facilitate getting over the initial hurdles of this culture change, tools that allow this transition are required.

**Comment by MikeKSmith** [comment]
Added these two blog posts following revisions. Will schedule these for posting.

File: content/post/But_I_only_use_NONMEM.Rmd (hunk 3 above)
Comment (kylebaron) [reply content/post/But_I_only_use_NONMEM.Rmd:99]: I'm not disagreeing with anything so far ... just bringing some nuance in the comparison with other fields: 

__Regarding comparison with progress in other fields__: do they have anything equivalent to regulatory agencies in their field?  I mean it's one thing to try some new tool or method in analyzing your data, writing it up as a manuscript, submitting it to the journal with the publication on the line.  But another thing to do the work and submit to regulatory agency with the approval on the line.  In the former case, you might be praised and rewarded for novelty and innovation and taking a risk.  In the latter case ... I don't know.  You might be opening yourself up to more scrutiny.  Maybe not.  

I get a lot of questions about how FDA views `mrgsolve`.   I can say we've used it in submission work without issue before.  It's not that they asked and said "ok" or they asked and had a lot of questions.  *No one has ever asked*.  So I just don't worry about it.  I suspect enough people working in industry still worry about it too much?  

I'm just saying that maybe the culture change needs to take into account the influence of regulators in our work (regardless of *where* you work) that might not be present in other fields.

Comment (vjd) [reply content/post/But_I_only_use_NONMEM.Rmd:99]: Great point @kylebmetrum ! To that point, my response would be the following -
Other than a few folks like yourself and Ron etc, people have not even figured out the possibilities. As a community, if we work together to bring tools into the open and work more collaboratively, and actually produce material with substance, then we can worry about how the regulators or others would accept. 
It comes down to pushing through these hurdles to show that good work can be done collaboratively

Comment (MikeKSmith) [reply content/post/But_I_only_use_NONMEM.Rmd:99]: Thanks for the input @kylebmetrum and @vjd. 

The point addressing "fear, uncertainty and doubt" around using open-source software should probably be addressed in a separate blog post! ;-)  I could write a whole separate post on that one. Maybe I should slim down text pertaining to that here?

The main point of this blog is really about breaking out of the mindset that the models we develop are only of interest *within* the pharmacometrics community and so there's no need for new language standards. The open-source bit is really the cherry on top, as the community itself can contribute, refine and evolve the language.

Comment (kylebaron) [reply content/post/But_I_only_use_NONMEM.Rmd:99]: @MikeKSmith Agreed.  My comment was a bit off topic with respect to the original blog post. ... more interacting with @vjd comment about comparison with other fields.

Comment (MikeKSmith) [reply content/post/But_I_only_use_NONMEM.Rmd:99]: Happy to receive a blog post on this topic anytime you like!! ;-)

**Pull request merged**